src/helm/benchmark/static/benchmarking.js

```diff
@@ -1186,96 +1180,120 @@ $(function () {
   //////////////////////////////////////////////////////////////////////////////
   const $main = $('#main');
   const $summary = $('#summary');
-  $.when(
```

Comment:
The way that Github is highlighting this is misleading -- there should be minimal changes `.when` and below from my code. It will be easier to view if you look at the last commit before August 19.